conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -9,7 +10,7 @@ services:
     hostname: des_redis
 
   rabbitmq:
-    image: rabbitmq:3.6.10-management
+    image: arm64v8/rabbitmq:3.6.14-management
```

Comment:
These docker-compose changes aren't relevant the metrics modal, so they should be removed.